ibm/service/satellite/resource_ibm_satellite_cluster.go

```diff
@@ -98,6 +98,11 @@ func ResourceIBMSatelliteCluster() *schema.Resource {
 				Computed:    true,
 				Description: "The lifecycle state of the cluster.",
 			},
+			"infrastructure_topology": {
+				Type:        schema.TypeString,
+				Optional:    true,
```

Comment:
As per description the default is "highly-available" so if user doesn't set this attribute but API gives back the "highly-available" then we may see diff in next plan commnads Either can we add "Computed" or "default"